travispy/entities/log.py

```diff
@@ -1,23 +1,38 @@
 from ._entity import Entity
 
+from travispy._helpers import get_archived_log
+
 
 class Log(Entity):
     '''
     :ivar int job_id:
         Jod ID.
 
-    :ivar str body:
-        Log body.
-
     :ivar str type:
     '''
 
     __slots__ = [
         'job_id',
-        'body',
+        '_body',
         'type',
     ]
 
+    def __init__(self, session):
```

Comment:
Nope, I think it is a good idea let it public. It is already exposed through the official API :smile: